.github/workflows/ci.yml

```diff
@@ -114,94 +123,41 @@ jobs:
       - name: Run Barge
         working-directory: ${{ github.workspace }}/barge
         run: |
-          export CONTRACTS_VERSION=v2.3.0
-          bash -x start_ocean.sh --no-aquarius --no-provider --no-dashboard --with-typesense 2>&1 > start_ocean.log &
-      - run: npm ci
-      - run: npm run build
-      - run: docker image ls
-      - name: Delete default runner images
-        run: |
-          docker image rm -f node:20
```

Comment:
Can we keep those docker image removals in the workflow if they exist? The runner pull those images by default and we do not need them.